docs/en/advanced-features/grpc-tls.md

```diff
@@ -0,0 +1,36 @@
+# Support Transport Layer Security (TLS)
+Transport Layer Security (TLS) is a very common security way when transport data through Internet.
+In some use cases, end users report the background:
+
+> Target(under monitoring) applications are in a region, which also named VPC,
+at the same time, the SkyWalking backend is in another region (VPC).
+>
+> Because of that, security requirement is very obvious.
+
+## Creating SSL/TLS Certificates
+
+The first step is to generate certificates and key files for encrypting communication. This is
+fairly straightforward: use `openssl` from the command line.
+
+Use this [script](../../../tools/TLS/tls_key_generate.sh) if you are not familiar with how to generate key files.
+
+We need the following files:
+- `client.pem`: A private RSA key to sign and authenticate the public key. It's either a PKCS#8(PEM) or PKCS#1(DER).
+- `client.crt`: Self-signed X.509 public keys for distribution.
+- `ca.crt`: A certificate authority public key for a client to validate the server's certificate.
+
+## Authentication Mode
+- Find `ca.crt`, and use it at client side. In `mTLS` mode, `client.crt` and `client.pem` are required at client side.
+- Find `server.crt`, `server.pem` and `ca.crt`. Use them at server side. Please refer to `gRPC Security` of the OAP server doc for more details.
+
+## Enable TLS
+- Enable (m)TLS on the OAP server side, [read more on this documentation](https://skywalking.apache.org/docs/main/v9.5.0/en/setup/backend/grpc-security/).
```

Comment:
I think `next` release doc may be a more stable link. 9.5.0 could be removed from doc list w/o updating here.

Reply:
update to the `next`